src/pages/installation-pending.js

```diff
@@ -15,13 +15,18 @@ You can also check in on the progress by contacting us at support@roadie.io or v
 const GithubAppInstallHoldingPage = () => (
   <>
     <SitewideHeader />
+
+    <div className="bg-white">
```

Comment:
Have to update these since I extracted the background color from the inside of the component.